env NOTEST does not seem to work, still seeing 

  Linking CXX executable test

in the output.

and linking 800mb executable, with embedding systemlibs to it, while no tests are performed, is making packaging really slow.
--- hhvm-HHVM-2.3.2/hphp/CMakeLists.txt~	2013-12-23 20:51:42.000000000 +0000
+++ hhvm-HHVM-2.3.2/hphp/CMakeLists.txt	2014-01-03 08:35:29.988356817 +0000
@@ -52,7 +52,7 @@
 
 add_subdirectory(hhvm)
 
-if (NOT "$ENV{HPHP_NOTEST}" STREQUAL "1")
+if (NOT HPHP_NOTEST)
 	add_subdirectory(test)
 endif ()
 
